packages/jest-expo/README.md

```diff
@@ -46,6 +58,76 @@ To test specific platforms you can use the following extensions:
 - web: `-test.web.*`
 - Node: `-test.node.*`, `-test.web.*`
 
+### Mixing runners
+
+If you don't want to use every runner you can always mix runners by using the `projects` field of your Jest config. This will only work with single-runner projects like `jest-expo/ios`, `jest-expo/android`, `jest-expo/web`, and `jest-expo/node`.
+
+```diff
+"jest": {
+-  "preset": "jest-expo/universal"
+// Skip web and node tests
++ "projects": [
++    { "preset": "jest-expo/ios" },
++    { "preset": "jest-expo/android"}
++ ]
+},
+```
+
+### Testing JSX Components
+
+To test the output of your React components you can use the [**Enzyme**](https://airbnb.io/enzyme/) testing library. `jest-expo` provides a convenience method for adding Enzyme to an existing Jest config (this is only supported with Jest configs provided by `jest-expo`). This implementation of Enzyme will also use a custom serializer for styles which should make snapshot tests cleaner and easier to read.
+
+#### Setup Enzyme
+
+1. Create a `jest.config.js` in your root directory and delete the `jest` field in your `package.json`
+2. Import the platforms you want to test and add enzyme support:
+
+```js
+// Skipping Node because components shouldn't be rendering in SSR
+const { getWebPreset, getIOSPreset, getAndroidPreset } = require('jest-expo/src/getPlatformPreset');
```

Comment:
we should export these things from `jest-expo` instead of encouraging people to reach into `src`